website/docs/r/pag_instance.html.markdown

```diff
@@ -0,0 +1,178 @@
+          "securitygroups" : [for sg in data.ibm_is_security_group.pag_instance_1 : sg.id],
+          "subnet" : {
+            "crn" : data.ibm_is_subnet.pag_instance_1.crn,
+            "cidr" : data.ibm_is_subnet.pag_instance_1.ipv4_cidr_block
+          }
+        },
+        {
+          "name" : "proxy2",
+          "securitygroups" : [for sg in data.ibm_is_security_group.pag_instance_2 : sg.id],
+          "subnet" : {
+            "crn" : data.ibm_is_subnet.pag_instance_2.crn,
+            "cidr" : data.ibm_is_subnet.pag_instance_2.ipv4_cidr_block
+          }
+        }
+      ]
+    }
+  )
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+}
+
+resource "ibm_iam_authorization_policy" "pag-cos-iam-policy" {
+  source_service_name         = "privileged-access-gateway"
+  source_resource_instance_id = ibm_pag_instance.pag.guid
+  roles                       = ["Object Writer"]
+  resource_attributes {
+    name     = "serviceName"
+    operator = "stringEquals"
+    value    = "cloud-object-storage"
+  }
+
+  resource_attributes {
+    name     = "accountId"
+    operator = "stringEquals"
+    value    = data.ibm_resource_group.pag.account_id
+  }
+  resource_attributes {
+    name     = "serviceInstance"
+    operator = "stringEquals"
+    value    = data.ibm_resource_instance.pag-cos.guid
+  }
+
+  resource_attributes {
+    name     = "resourceType"
+    operator = "stringEquals"
+    value    = "bucket"
+  }
+
+  resource_attributes {
+    name     = "resource"
+    operator = "stringEquals"
+    value    = var.ibm_cos_bucket_name
+  }
+
+}
+
+
+locals {
+  pag_hostnames = [for i in range(var.num_instances) : join(".", ["${ibm_pag_instance.pag.guid}-${i + 1}", "${ibm_pag_instance.pag.location}", "pag", "appdomain", "cloud"])]
+}
+output "pag-hosts" {
+  value = local.pag_hostnames
+}
+```
+
+
+
+The `ibm_resource_instance` resource provides the following [Timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:
+
+- **create** - (Default 15 minutes) Used for Creating Instance.
+- **update** - (Default 15 minutes) Used for Updating Instance.
+- **delete** - (Default 15 minutes) Used for Deleting Instance.
+
+## Argument reference
+Review the argument references that you can specify for your resource. 
+
+- `location` - (Required, String) Target location or environment to create the PAG instance.
+- `parameters_json` (Required ,String) Parameters to create PAG instance. The value must be a JSON string.
+- `plan` - (Required, String) The name of the plan type supported by service.
+- `name` - (Required, String) A descriptive name used to identify the resource instance.
+- `resource_group_id` - (Required, String) The ID of the resource group where you want to create the PAG service. You can retrieve the value from data source `ibm_resource_group`. If not provided creates the service in default resource group.
+- `service` - (Required, String) The name of the service.
```

Comment:
The service name also can we add wt value it accepts since its contant and remains same.

Reply:
This is addressed.